packages/frontend/src/components/ElectionForm/Details/ElectionDetailsInlineForm.tsx

```diff
@@ -23,34 +24,34 @@ export default function ElectionDetailsInlineForm() {
 
     return (
         <Paper elevation={3}>
+        <>
             {!open &&
                 <Grid container
                     sx={{
                         m: 0,
-                        p: 1,
+                        p: 4,
                     }}
                 >
-                    <Grid xs={12}>
-                        <Typography gutterBottom variant="h4" component="h4">
-                            Election Title: {election.title}
-                        </Typography>
+                    <Grid item container xs={11}>
+                    <Grid item xs={12}>
+                                <Typography variant="h3" component="h4">
+                                    {election.title}
+                                    <ElectionStateChip state={election.state} />
+                                </Typography>
+                            </Grid>
+                        {election.description && <Grid xs={12}>
+                            <Typography gutterBottom variant="h6" component="h5">
+                                {election.description}
+                            </Typography>
+                        </Grid>}
+                        {(election.start_time || election.end_time) && 
+                            <Grid xs={12}>
+                            <Typography sx={{mt: 2}} component="p" variant='subtitle2'>
+                                {election.start_time ? formatDate(election.start_time, election.settings.time_zone) : 'none'} - {election.end_time ? formatDate(election.end_time, election.settings.time_zone) : 'none'}
```

Comment:
I like this format when both are defined, but I think "none - August 13th 2024" could look a bit awkward. 

Maybe it's worth breaking this into separate cases, and you can do "starts at <date>" or "ends at <date>" whenever only one of them is defined?

Reply:
Ok, I updated it to dynamically display the date base on if one or both is set.